app/controllers/ApplicationController.scala

```diff
@@ -142,6 +145,20 @@ class ApplicationController @Inject()(
     }
   }
 
+  def makePayment: Action[AnyContent] = VerifiedAction(baseBreadcrumb) { implicit pertaxContext =>
```

Comment:
Is this the best place for this method. I'm just think that `ApplicationController` is a very generic name and maybe this would be better placed in a `PaymentsController` or similar